addons/stock/migrations/11.0.1.1/post-migration.py

```diff
@@ -188,7 +188,7 @@ def create_stock_move_line(env):
             LEFT JOIN stock_production_lot spl ON spl.id = sq.lot_id
         WHERE sm.state = 'done'
         GROUP BY sq.lot_id, spo.product_id, spo.owner_id, spo.package_id,
-            spo.result_package_id""",
+            spo.result_package_id, spo.picking_id""",
```

Comment:
You should group by `sm.id`, not by picking, but indeed here we need less grouping.